analytics/testapp/ProjectSettings/ProjectSettings.asset

```diff
@@ -183,7 +183,7 @@ PlayerSettings:
     serializedVersion: 2
     m_Bits: 238
   iPhoneSdkVersion: 988
-  iOSTargetOSVersionString: 10.0
+  iOSTargetOSVersionString: 11.0
```

Comment:
I kind of feel that we should remove this line and use whatever default value for given Unity version. 

1. We should make sure that Firebase Unity SDK + EDM4U works with a freshly created Unity project in any given Unity version.
2. Ppl who downloaded this testapp maybe using older version of Unity SDK which only required iOS SDK `10.0`.
3. Updating value here for the test project kind of hide the potential issue in `Dependencies.xml` or perhaps bugs in EDM4U...

On the other hand, I think this is ok as a temporary workaround to make our GHA green before we fixes `Dependencies.xml` and release it.

Reply:
The GHA is not affected since CI is not pulling this repo for testing. This is to reduce the breakage and confusion when developers are using this repo as the start app. And the long run fix is already tracked as an issue.